README.md

```diff
@@ -193,6 +193,38 @@ In the case one of your dependencies is installing `babel` and you cannot uninst
   }
 ```
 
+## Customize config for specific Webpack target
+
+Webpack supports bundling for multiple different [targets](https://webpack.js.org/concepts/targets/) including the web, and node. Sometimes it can be useful to have a slightly different babel configuration for the different environments. This loader provides a `target` property via the babel [caller](https://babeljs.io/docs/en/config-files#apicallercb) API which can be used in this case:
+
+```javascript
+// babel.config.js
+
+module.exports = api => {
+  return {
+    plugins: [
+      "@babel/plugin-proposal-nullish-coalescing-operator",
+      "@babel/plugin-proposal-optional-chaining"
+    ],
+    presets: [
+      [
+        "@babel/preset-env",
+        {
+          useBuiltIns: "entry",
+          // caller.target will be the same as the target option from webpack
+          ...(api.caller(caller => caller.target === "node")
+            ? { node: "current" }
+            : { targets: { chrome: "58", ie: "11" } }
+          )
+        }
+      ]
+    ]
+  }
+}
+```
+
+In the above `babel.config.js` we're able to distinguish if we are transpiling for `node` or the `web` and adjust accordingly. In this case it has been configured such that `node` targets use `@babel/preset-env` to transpile features for the current version of node, while the `web` target will transpile features back to ie 11 and Chrome 58.
```

Comment:
I feel like we can skip explaining the targets with a quick description of the example (maybe with something like what I suggested above)?

Thoughts?

Reply:
I agree. I have incorporated these changes in the PR. Thanks for the feedback 😄